Swiftfin/Views/SettingsView/PlaybackQualitySettingsView.swift

```diff
@@ -44,18 +57,19 @@ struct PlaybackQualitySettingsView: View {
                 } header: {
                     L10n.bitrateTest.text
                 } footer: {
-                    VStack(alignment: .leading, spacing: 8) {
-                        L10n.bitrateTestDescription.text
-                        L10n.bitrateTestDisclaimer.text
+                    VStack(alignment: .leading) {
+                        Text(L10n.bitrateTestDisclaimer)
+                        LearnMoreButton(L10n.bitrateTest) {
```

Comment:
`Learn More` should be for more detail on multiple items or for a single section that requires a lot more description. This specific section can be described concisely, so we don't need a `Learn More` for it.